torch_points3d/core/data_transform/grid_transform.py

```diff
@@ -118,7 +118,7 @@ def _process(self, data):
         if "batch" not in data:
             cluster = grid_cluster(coords, torch.tensor([1, 1, 1]))
         else:
-            cluster = voxel_grid(coords, data.batch, 1)
+            cluster = voxel_grid(coords, 1, data.batch)
```

Comment:
And, this was a breaking change related to `torch-geometric`, argument order was different in the new version - causing failures on the IOS side. Fixed with this one line change.

Reply:
Nice catch :+1:  That's a minor change in their logs, non-breaking by changing arg order :upside_down_face:

Reply:
Yeah, spent some time into debugging this, luckily it wasn't that difficulty 😅